src/cameraGeom/Detector.cc

```diff
@@ -212,7 +243,7 @@ class DetectorFactory : public table::io::PersistableFactory {
 
     std::shared_ptr<table::io::Persistable> read(InputArchive const& archive,
                                                  CatalogVector const& catalogs) const override {
-        auto const & keys = PersistenceHelper::get();
+        auto const & keys = PersistenceHelper(catalogs.front().getSchema());
```

Comment:
While C++ lets you get away with binding a temporary to a const reference, for a local variable like this I think it's better to just assign it to a value (i.e. `auto const` instead of `auto const &`).

Reply:
Cargo-culted from the previous lone, which worked because it was a singleton.  I'll change it